source/js/primary-nav.js

```diff
@@ -20,6 +20,20 @@ let primaryNav = {
       }
     }
 
+    const summaryBar = document.querySelector(".article-navbar");
+    if(summaryBar) {
+      const heroHeight = document.querySelector('.publication-hero-container').offsetHeight;
+      const navHeight = document.querySelector(".primary-nav-container").offsetHeight;
+      const total = heroHeight + navHeight - summaryBar.offsetHeight;
+      window.addEventListener("scroll", () => {
+        if(window.scrollY > total) {
+          summaryBar.style.marginTop = "0px";
```

Comment:
Unitless `0` should just work.
```suggestion
          summaryBar.style.marginTop = "0";
```